src/generators/java/presets/CommonPreset.ts

```diff
@@ -110,15 +182,26 @@ export const JAVA_COMMON_PRESET: JavaPreset = {
       const shouldContainEqual = options.equal === undefined || options.equal === true;
       const shouldContainHashCode = options.hashCode === undefined || options.hashCode === true;
       const shouldContainToString = options.classToString === undefined || options.classToString === true;
+      const shouldContainMarshal = options.marshal === true;
```

Comment:
@ritik307 try double-checking what options you pass in your test and see if it matches this one 😄

Reply:
@jonaslagoni  Sorry to bother you again but I am not able to figure out where to check for the options :(

Reply:
Don't worry about it, what I mean by options, is the `JavaCommonPresetOptions`, and what was the property you added there vs what are you using here 🙂

Reply:
Hint, look at https://github.com/asyncapi/modelina/pull/596/files#diff-a1e439279cc6c3345bcb667a54b404bc408869de67235c3133af054088cfe854R12